styles/Fly/Latin.yml

```diff
@@ -0,0 +1,13 @@
+extends: substitution
+message: "Use '%s' instead of '%s'."
+ignorecase: true
+level: warning
+nonword: true
+action:
+  name: replace
+swap:
+  '\b(?:eg|e\.g\.)(?=[\s,;])': for example
+  '\b(?:ie|i\.e\.)(?=[\s,;])': that is
```

Comment:
Both the Google Developer Documentation Style Guide and the Microsoft Writing Style Guide have these abbreviations on their list of words not to use. This might stem from a push for simplified technical English and/or from requirements for localization of text.
We can decide to keep using them if that's the preference.